fmriprep/workflows/bold.py

```diff
@@ -1017,6 +1024,36 @@ def select_target(subject_id, space):
         iterables=('hemi', ['lh', 'rh']),
         name='sampler')
 
+    def medial_wall_to_nan(in_file, subjects_dir):
+        """ Convert values on medial wall to NaNs
+        """
+
+        import nibabel as nb
+        import numpy as np
+        import os
+    
+        fn = os.path.basename(in_file)
+        space = fn.split('.')[1]
+
+        if not space.startswith('fsaverage'):
+            return in_file
```

Comment:
Is this simply because `fsnative` isn't a subject, or is it because you don't want to set medial walls to NaN in `fsnative`?

If the former, you can use the output of `select_target` to get the actual subject name.

Reply:
I actually wasn't sure whether anyone would want that output - but I've added it in, thanks.